### PR TITLE
Fixes #28214 - lock all input fields

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by ares** [comment]
please see the original discussion at https://github.com/theforeman/foreman/pull/7418, I was unable to reopen that one, assigning @tbrisker for the review

File: app/views/template_inputs/_form.html.erb [hunk 1]
```diff
@@ -3,9 +3,9 @@
...
-      <%= selectable_f f, :input_type, template_input_types_options(@template.acceptable_template_input_types), {}, :class => 'input_type_selector without_select2', :disabled => @template.locked? %>
-      <%= selectable_f f, :value_type, [[_('Plain'), 'plain'], [_("Search"), 'search'], [_('Date'), 'date']], {}, {'data-item': f.index || f.object.id, class: 'select-value-type', onchange: "tfm.templateInputs.inputValueOnchange(this)"}  %>
-      <%= selectable_f f, :resource_type, Permission.resources_with_translations, {}, {wrapper_class: "#{hide_resource_type_input(f.object)} form-group resource-type-#{f.index || f.object.id}"} %>
+      <%= selectable_f f, :input_type, template_input_types_options(@template.acceptable_template_input_types), {}, :class => "input_type_selector #{@template.locked? && 'without_select2'}", :disabled => @template.locked? %>
+      <%= selectable_f f, :value_type, [[_('Plain'), 'plain'], [_("Search"), 'search'], [_('Date'), 'date']], {}, {'data-item': f.index || f.object.id, class: "select-value-type #{@template.locked? && 'without_select'}", onchange: "tfm.templateInputs.inputValueOnchange(this)", :disabled => @template.locked? } %>
```
Comment (tbrisker) [inline app/views/template_inputs/_form.html.erb]:
```suggestion
      <%= selectable_f f, :value_type, [[_('Plain'), 'plain'], [_("Search"), 'search'], [_('Date'), 'date']], {}, {'data-item': f.index || f.object.id, class: "select-value-type #{@template.locked? && 'without_select2'}", onchange: "tfm.templateInputs.inputValueOnchange(this)", disabled: @template.locked? } %>
```

File: app/views/template_inputs/_form.html.erb [hunk 2]
```diff
@@ -3,9 +3,9 @@
...
+      <%= selectable_f f, :resource_type, Permission.resources_with_translations, {}, {wrapper_class: "#{hide_resource_type_input(f.object)} form-group #{@template.locked? && 'without_select2'} resource-type-#{f.index || f.object.id}", :disabled => @template.locked? } %>
```
Comment (tbrisker) [inline app/views/template_inputs/_form.html.erb]:
```suggestion
      <%= selectable_f f, :resource_type, Permission.resources_with_translations, {}, {wrapper_class: "#{hide_resource_type_input(f.object)} form-group resource-type-#{f.index || f.object.id}", class: "#{@template.locked? && 'without_select2'}", disabled: @template.locked? } %>
```

**Review by tbrisker — CHANGES_REQUESTED** [review]

**Comment by tbrisker** [comment]
what's the status here @ares ?

**Comment by ares** [comment]
my applogies, accepted your suggestions, hopefully that does not disqualify you from merging :-)

**Comment by ares** [comment]
rebased to run tests again, simple patch to be merged

**Review by tbrisker — CHANGES_REQUESTED** [review]
Looks like you resolved the proposed changes but didn't apply them, maybe they got lost in the rebase?

**Comment by ares** [comment]
sorry, commited both suggestions, not sure where it got lost

**Review by tbrisker — APPROVED** [review]
Thanks @ares ! tested, works as expected.

**Comment by tbrisker** [comment]
and fixed the commit message on merge for you

**Comment by tbrisker** [comment]
2.3 - 11ac0615d

**Pull request merged**